### PR TITLE
Update docs on create Router docker image with interceptor lua changes

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Purpose
$subject

### Issues

Doc

### Automation tests
 - Unit tests added: N/A
 - Integration tests added: N/A

### Tested environments

Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: router/src/main/resources/interceptor/README.md [hunk 1]
```diff
@@ -39,3 +41,23 @@
...
+
+## Build Router Docker Image - for Integration/Local Testing
+
+Set `ROUTER_BASE_IMAGE` with the latest compatible Router docker image and `TARGET_ROUTER_IMAGE` with the image name that you want to be built.
+
+For example:
+- ROUTER_BASE_IMAGE=wso2/choreo-connect-router:1.1.0
+- TARGET_ROUTER_IMAGE=wso2/choreo-connect-router:1.2.0-m1-SNAPSHOT
+
+Change directory to here and execute the following script.
```
Comment (suksw) [inline router/src/main/resources/interceptor/README.md:53]:
```suggestion
Change directory to here (to `router/src/main/resources/interceptor`) and execute the following script.
```

File: router/src/main/resources/interceptor/README.md [hunk 2]
```diff
@@ -39,3 +41,23 @@
...
+
+```shell
+cat <<EOF > DockerfileAddInterceptor
+FROM $ROUTER_BASE_IMAGE
+COPY interceptor /home/wso2/interceptor
+EOF
+
+docker build -t $TARGET_ROUTER_IMAGE -f DockerfileAddInterceptor ../
```
Comment (suksw) [inline router/src/main/resources/interceptor/README.md:61]: Is it `./`?
```suggestion
docker build -t $TARGET_ROUTER_IMAGE -f DockerfileAddInterceptor ./
```

Comment (renuka-fernando) [reply router/src/main/resources/interceptor/README.md]: fixed with https://github.com/wso2/product-microgateway/pull/2978/commits/4b95f76810ac4716491351e69c9129084553f61d

File: router/src/main/resources/interceptor/README.md (hunk 1 above)
Comment (renuka-fernando) [reply router/src/main/resources/interceptor/README.md]: fixed with https://github.com/wso2/product-microgateway/pull/2978/commits/4b95f76810ac4716491351e69c9129084553f61d

**Review by suksw — APPROVED** [review]

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**